### ISSUE TITLE
Possible way of passing the initial extrinsic parameters

### ISSUE THREAD

**Issue by MikeZhu92** [tudelft-iv/multi_sensor_calibration#6]
Just curious if it's possible to pass the initial guess of the extrinsic parameters.
After testing the bag, I realize the lidar and mono calibration solution is not bounded enough to get the corrected results.

Any tips for where I should look into the code would be great.

**Comment by EaiPool** [comment]
Indeed, this is currently not possible. Possible places to implement this for each method is [here](https://github.com/tudelft-iv/multi_sensor_calibration/blob/master/optimization/src/optimization/PSE.py#L87), [here](https://github.com/tudelft-iv/multi_sensor_calibration/blob/master/optimization/src/optimization/MCPE.py#L100) and [here](https://github.com/tudelft-iv/multi_sensor_calibration/blob/fa0a130512ab17f033683a66a84ef932b7f964a9/optimization/src/optimization/FCPE.py#L75).

However, in most situations it should be possible to automatically generate a pretty alright initial solution. How many calibration board locations are you using? The [documentation](https://github.com/tudelft-iv/multi_sensor_calibration#record-calibration-board-locations-and-perform-calibration) suggests to use at least 10.
